.travis.yml

```diff
@@ -47,6 +51,7 @@ script:
   - composer phpunit-sf
   - composer phpunit-controllers
   - bash ./travis-scripts/run-selenium-tests
+  - bash ./travis-scripts/run-functionnal-tests
```

Comment:
Has to be executed before starter theme, because in this script the default theme is removed and replaced by another one